### DISCUSSION TITLE
Bullet Journal features

### DISCUSSION THREAD

**Discussion by vicrdguez** [vicrdguez/dendroam#9] [Ideas]
I think it could be useful to implement a simple bullet journal workflow to manage daily tasks 
using dailies.

The Idea is to take all open *TODO* headings from the past day when you open 
your daily journal note for today. Instead of deleting old *TODO* entries, we can 
use something like *MOVED*, writing the date when the task were refiled.

This is an idea more minimal than org agenda, that could allow more focus at the 
tasks you must accomplish for today. 

This can be extended to weekly notes and monthly notes as well, so organizing TODOs 
will be more based on time, potentially avoiding the _Collector Fallacy_.

Using org agenda is still possible and looking 
at org-roam code including todos in the db, actually a future integration 
with agenda could be easy, showing you just your notes for today, for the rest of 
the week or the rest of the month, instead of showing just all possible tasks that
you might have in your agenda files.

**Comment by theianjones** [comment]
I like this idea quite a bit. Right now I have to put everything in a `todo.org` because I want to see my daily tasks in my org agenda but dont want to add a bunch of files to be sources

Heres how `org-journal` does it. I like the modification that the old todos get `MOVED`. I dont think org journal does this 

https://github.com/bastibe/org-journal#carry-over

**Reply by vicrdguez** [reply]
Actually I have the exact same problem, I got inspired for org-journal as well. 
What I was thinking is something simple that does not require to much interaction for the user to manage your dailies, weeklies and monthlies as well.

The `MOVED` thing is something I personally find important, because if a task gets migrated I don't want to loose track of when it was created, so it is easier to see which task are you procrastinating on and maybe should be dropped. That's something that Bullet Journal makes really great.

**Reply by codeluggage** [reply]
I currently do this by refiling and having `(setq org-log-refile (quote time))` set - did not get around to making it also record where it was refiled from, but the timestamp does give an indication. Having automation around this for dailies would be nice!
